src/main.cpp

```diff
@@ -78,6 +158,7 @@ int main ( int argc, char** argv )
 #else
     bool bIsClient = true;
 #endif
+    bool         bSpecialOptions             = false; // Any options after this option will be accepted ! (mostly used for debugging purpouses)
```

Comment:
What's that exactly? Is it a new debugging feature?

Reply:
> What's that exactly? Is it a new debugging feature?

Yes since we now can access the command line parameters anywhere in the code those are also very useful for debugging and testing purposes too. But since there is a check in main which would accept only valid, known parameters I introduced the "--special" option to allow any option for debugging and test purposes.